src/features/tweaks/subtle_activity_mutuals.js

```diff
@@ -3,9 +3,10 @@ import { keyToCss } from '../../utils/css_map.js';
 import { buildStyle } from '../../utils/interface.js';
 import { dom } from '../../utils/dom.js';
 
-const labelSelector = `${keyToCss('followingBadgeContainer', 'mutualsBadgeContainer')}:has(> svg)`;
+const labelSelector = `${keyToCss('followingBadgeContainer', 'mutualsBadgeContainer')}:is(${keyToCss('activity', 'activityItem')} *)`;
```

Comment:
i think this can be simplified?

i was wondering why i never thought of using `:is()` to filter to parent selectors, when i realised, wait, CSS just lets you do that without any tricks

```suggestion
const labelSelector = `${keyToCss('activity', 'activityItem')} ${keyToCss('followingBadgeContainer', 'mutualsBadgeContainer')}`;
```

Reply:
Oh yeah. I've absolutely done it on purpose for readability reasons when composing selectors, but I have no idea why I did it here :D